ext/XS-APItest/t/locale.t

```diff
@@ -118,7 +118,7 @@ SKIP: {
         chomp;
         next unless / - \d+ $ /x;
         s/ ^ \# \s* define \s*//x;
-        m/ (.*) \  (.*) /x;
+        m/ (\S+) \s+ (.*) /x;
```

Comment:
I updated the commit message. I suspect you are right, but the code in practice doesnt have such things. It was just that if you changed a define from 'define foo bar' into 'define foo   bar' the parse code would see the define name as 'foo    ' and then fail. Such whitespace sensitivity is unhelpful. :-) And as you said, had it had a more complex value it would have gotten things worse.